docs/api-reference/next.config.js/environment-variables.md

```diff
@@ -4,6 +4,8 @@ description: Learn to add and access environment variables in your Next.js appli
 
 # Environment Variables
 
+> Since the release of Next.js 9.4 we now have a more intuitive and ergonomic experience for [adding environment variables](/docs/basic-features/environment-variables.md), give it a try!
```

Comment:
```suggestion
> Since the release of Next.js 9.4 we now have a more intuitive and ergonomic experience for [adding environment variables](/docs/basic-features/environment-variables.md). Give it a try!
```